go.mod

```diff
@@ -10,12 +10,12 @@ require (
 	github.com/fatih/color v1.9.0
 	github.com/gin-contrib/cors v1.3.0
 	github.com/gin-contrib/gzip v0.0.2-0.20200226035851-25bef2ef21e8
-	github.com/gin-contrib/sessions v0.0.1
+	github.com/gin-contrib/sessions v0.0.5
```

Comment:
Why do we need to upgrade those pkgs?

Reply:
Older versions of the package do not provide the `SameSite` parameter